.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
What is this used for?

Reply:
File created from IDE.  File has been removed.